src/Microsoft.Diagnostics.DebugServices.Implementation/CommandService.cs

```diff
@@ -382,78 +383,73 @@ internal void CreateCommand(Type type, CommandAttribute commandAttribute, Func<I
 
                 foreach (string alias in commandAttribute.Aliases)
                 {
-                    command.AddAlias(alias);
+                    command.Aliases.Add(alias);
```

Comment:
I haven't looked at it but when test your latest changes, command alases don't work.  Like "q" for the "exit" for example.

Reply:
@mikem8361 thanks! I've fixed the aliases but it keeps failing.

I was able to do some debugging and I got to a weird point: when I don't have the VS debugger attached I am getting following error:

![{B15B6F07-8378-4FD2-92FB-F02B317C79B1}](https://github.com/user-attachments/assets/a889b471-7478-47e8-86f6-7416a5523bdf)

When I edit `src/Tools/dotnet-dump/Analyzer.cs` and add `System.Diagnostics.Debugger.Launch();` and connect with the VS debugger, I can see everything being populated, but at the end I am getting a different error (it seems that the command was parsed fine, but it has simply failed for some reason):

![{8F18993B-6E64-49F8-A413-3BAE5AAF02D5}](https://github.com/user-attachments/assets/06e55ec1-1d41-4ead-95b9-afbb8f6673bb)

I'll try to wrap my head around it

Reply:
I think this is fixed in main.  Merge/rebase main.